### PR TITLE
adding in additional column to vindex table

### LINKED ISSUE #713: Document ignore_nulls for lookup vindexes
From https://github.com/vitessio/vitess/issues/6147 , https://github.com/vitessio/vitess/pull/6222

### PR DESCRIPTION
**Pull request by hallaroo**
For easy preview: https://deploy-preview-714--vitess.netlify.app/docs/reference/features/vindexes/#ignore-nulls

### CODE REVIEW

**Comment by hallaroo** [comment]
Note still need to confirm add nulls (consistent look up and lookup)

File: content/en/docs/reference/features/vindexes.md [hunk 1]
```diff
@@ -149,24 +149,24 @@ If the table owns lookup vindexes, then the rows to be deleted are first read an
...
+| null | Functional Unique | Always map to keyspace ID 0 | Yes | No | Yes | 100 | Any |
+| numeric | Functional Unique | Identity | Yes | Yes | No | 0 | 64 bit or smaller numeric or equivalent type |
+| numeric\_static\_map | Functional Unique | JSON file statically mapping input string values to keyspace IDs | Yes | No | No | 1 | Any |
+| region\_experimental | Functional Unique | Multi-column prefix-based hash for use in geo-partitioning | Yes | No | No | 1 | String and numeric type |
+| region\_json | Functional Unique | Multi-column prefix-based hash combined with a JSON map for key-to-region mapping, for use in geo-partitioning | Yes | No | No | 1 | String and numeric type |
+| reverse\_bits | Functional Unique | Bit reversal | Yes | Yes | Yes | 1 | 64 bit or smaller numeric or equivalent type |
+| unicode\_loose\_md5 | Functional Unique | Case-insensitive (UCA level 1) MD5 hash | Yes | No | Yes | 1 | String or binary types |
+| unicode\_loose\_xxhash | Functional Unique | Case-insensitive (UCA level 1) xxHash64 hash | Yes | No | Yes | 1 | String or binary types |
+| xxhash | Functional Unique | xxHash64 hash | Yes | No | No | 1 | Any |
```
Comment (hallaroo) [inline content/en/docs/reference/features/vindexes.md]: xxhash should be yes

File: content/en/docs/reference/features/vindexes.md [hunk 2]
```diff
@@ -149,24 +149,24 @@ If the table owns lookup vindexes, then the rows to be deleted are first read an
...
-| numeric\_static\_map | Functional Unique | JSON file statically mapping input string values to keyspace IDs | Yes | No | 1 | Any |
-| region\_experimental | Functional Unique | Multi-column prefix-based hash for use in geo-partitioning | Yes | No | 1 | String and numeric type |
-| region\_json | Functional Unique | Multi-column prefix-based hash combined with a JSON map for key-to-region mapping, for use in geo-partitioning | Yes | No | 1 | String and numeric type |
-| reverse\_bits | Functional Unique | Bit reversal | Yes | Yes | 1 | 64 bit or smaller numeric or equivalent type |
-| unicode\_loose\_md5 | Functional Unique | Case-insensitive (UCA level 1) MD5 hash | Yes | No | 1 | String or binary types |
-| unicode\_loose\_xxhash | Functional Unique | Case-insensitive (UCA level 1) xxHash64 hash | Yes | No | 1 | String or binary types |
-| xxhash | Functional Unique | xxHash64 hash | Yes | No | 1 | Any |
+| Name | Type | Description | Primary | Reversible | Nullable | Cost | Data types |
+| :--------------------- | ---- | ----------------------- | ------- | -------- | ------- | ---- | ---------- |
+| binary | Functional Unique | Identity | Yes | Yes | No | 0 | Any |
```
Comment (hallaroo) [inline content/en/docs/reference/features/vindexes.md]: binary should be yes

File: content/en/docs/reference/features/vindexes.md [hunk 3]
```diff
@@ -149,24 +149,24 @@ If the table owns lookup vindexes, then the rows to be deleted are first read an
...
+| binary\_md5 | Functional Unique | MD5 hash | Yes | No | No | 1 | Any |
```
Comment (hallaroo) [inline content/en/docs/reference/features/vindexes.md]: binary\_md5 should be yes

File: content/en/docs/reference/features/vindexes.md [hunk 4]
```diff
@@ -149,24 +149,24 @@ If the table owns lookup vindexes, then the rows to be deleted are first read an
...
+| hash | Functional Unique | DES null-key hash | Yes | Yes | No | 1 | 64 bit or smaller numeric or equivalent type |
+| lookup | Lookup NonUnique | Lookup table non-unique values | No | No | Yes | 20 | Any |
+| lookup\_unique | Lookup Unique | Lookup table unique values | If unowned | No | Yes | 10 | Any |
+| null | Functional Unique | Always map to keyspace ID 0 | Yes | No | Yes | 100 | Any |
+| numeric | Functional Unique | Identity | Yes | Yes | No | 0 | 64 bit or smaller numeric or equivalent type |
+| numeric\_static\_map | Functional Unique | JSON file statically mapping input string values to keyspace IDs | Yes | No | No | 1 | Any |
+| region\_experimental | Functional Unique | Multi-column prefix-based hash for use in geo-partitioning | Yes | No | No | 1 | String and numeric type |
+| region\_json | Functional Unique | Multi-column prefix-based hash combined with a JSON map for key-to-region mapping, for use in geo-partitioning | Yes | No | No | 1 | String and numeric type |
+| reverse\_bits | Functional Unique | Bit reversal | Yes | Yes | Yes | 1 | 64 bit or smaller numeric or equivalent type |
```
Comment (hallaroo) [inline content/en/docs/reference/features/vindexes.md]: reverse bits should be no

File: content/en/docs/reference/features/vindexes.md [hunk 5]
```diff
@@ -149,24 +149,24 @@ If the table owns lookup vindexes, then the rows to be deleted are first read an
...
-| region\_json | Functional Unique | Multi-column prefix-based hash combined with a JSON map for key-to-region mapping, for use in geo-partitioning | Yes | No | 1 | String and numeric type |
-| reverse\_bits | Functional Unique | Bit reversal | Yes | Yes | 1 | 64 bit or smaller numeric or equivalent type |
-| unicode\_loose\_md5 | Functional Unique | Case-insensitive (UCA level 1) MD5 hash | Yes | No | 1 | String or binary types |
-| unicode\_loose\_xxhash | Functional Unique | Case-insensitive (UCA level 1) xxHash64 hash | Yes | No | 1 | String or binary types |
-| xxhash | Functional Unique | xxHash64 hash | Yes | No | 1 | Any |
+| Name | Type | Description | Primary | Reversible | Nullable | Cost | Data types |
+| :--------------------- | ---- | ----------------------- | ------- | -------- | ------- | ---- | ---------- |
+| binary | Functional Unique | Identity | Yes | Yes | No | 0 | Any |
+| binary\_md5 | Functional Unique | MD5 hash | Yes | No | No | 1 | Any |
+| consistent\_lookup | Lookup NonUnique | Lookup table non-unique values | No | No | Yes | 20 | Any |
```
Comment (hallaroo) [inline content/en/docs/reference/features/vindexes.md]: Need to add qualification that ignore nulls on enables it to not fail 
It depends on if the lookup table primary vindex is nullable

File: content/en/docs/reference/features/vindexes.md [hunk 6]
```diff
@@ -149,24 +149,24 @@ If the table owns lookup vindexes, then the rows to be deleted are first read an
...
+| consistent\_lookup\_unique | Lookup Unique | Lookup table unique values | If unowned | No | Yes | 10 | Any |
```
Comment (hallaroo) [inline content/en/docs/reference/features/vindexes.md]: It depends on if the lookup table primary vindex is nullable

File: content/en/docs/reference/features/vindexes.md [hunk 7]
```diff
@@ -149,24 +149,24 @@ If the table owns lookup vindexes, then the rows to be deleted are first read an
...
+| hash | Functional Unique | DES null-key hash | Yes | Yes | No | 1 | 64 bit or smaller numeric or equivalent type |
+| lookup | Lookup NonUnique | Lookup table non-unique values | No | No | Yes | 20 | Any |
```
Comment (hallaroo) [inline content/en/docs/reference/features/vindexes.md]: The same as consistent lookup

File: content/en/docs/reference/features/vindexes.md [hunk 8]
```diff
@@ -149,24 +149,24 @@ If the table owns lookup vindexes, then the rows to be deleted are first read an
...
+| lookup\_unique | Lookup Unique | Lookup table unique values | If unowned | No | Yes | 10 | Any |
```
Comment (hallaroo) [inline content/en/docs/reference/features/vindexes.md]: The same as consistent lookup

**Review by aquarapid — APPROVED** [review]

**Pull request merged**